### ISSUE TITLE
pbsmrtpipe templates not available post-build

### ISSUE THREAD

**Issue by zachcp** [PacificBiosciences/pitchfork#406]
Hello Team Pitchfork,

I've just installed pbsmrtpipe  on a fresh ubuntu machine using pitchfork but I do not see any of the pipelines.  Is there  a way to register the pipelines?

Thank You,
zach cp

cc @mpkocher 

```
ubuntu@ip---:/data/WorkDir/OCT17_A1$ source ~/software/pitchfork/deployment/setup-env.sh

ubuntu@ip---:/data/WorkDir/OCT17_A1$ pbsmrtpipe show-template-details pbsmrtpipe.pipelines.polished_falcon_fat -o preset_tasks_falcon_fat.xml
Registry Loaded. Number of ToolContracts:39 FileTypes:67 ChunkOperators:2 Pipelines:18
Unable to find template id 'pbsmrtpipe.pipelines.polished_falcon_fat' in registered pipelines. Use the show-templates option to get a list of workflow options.

ubuntu@ip---:/data/WorkDir/OCT17_A1$ pbsmrtpipe show-templates
Run with --show-all to display (unsupported) developer/internal pipelines

ubuntu@ip---:/data/WorkDir/OCT17_A1$ pbsmrtpipe show-templates --show-all


**********************************************************************
18 Registered Developer/Internal Pipelines (name -> version, id, tags)
**********************************************************************
  1. Dev Hello Distributed Workflow Pipeline          0.1.0    pbsmrtpipe.pipelines.dev_dist                   cluster,dev
  2. Dev Local Hello Chunkable Pipeline               0.1.0    pbsmrtpipe.pipelines.dev_local_chunk            chunk,local,dev
  3. Dev Local Hello Pipeline                         0.1.0    pbsmrtpipe.pipelines.dev_local                  local,dev
  4. Dev Task Option Support Pipeline                 0.1.0    pbsmrtpipe.pipelines.dev_task_options           local,dev
  5. Dev local Task for Chunking pipelines            0.1.0    pbsmrtpipe.pipelines.dev_local_fasta_chunk      chunking,dev
  6. Example Dev 01 Subread DataSet pipeline          0.1.0    pbsmrtpipe.pipelines.dev_01_ds                  dev,reports
  7. Example Dev 01 crashing pipeline                 0.1.0    pbsmrtpipe.pipelines.dev_01_fail                dev
  8. Example Dev 01 pipeline                          0.1.0    pbsmrtpipe.pipelines.dev_01                     dev
  9. Example Dev 02 pipeline                          0.1.0    pbsmrtpipe.pipelines.dev_02                     dev
 10. Example Dev 03 pipelines                         0.1.0    pbsmrtpipe.pipelines.dev_03                     dev
 11. Example Dev 04                                   0.1.0    pbsmrtpipe.pipelines.dev_04                     dev
 12. Pipeline that leverages a python static tasks    0.1.0    pbsmrtpipe.pipelines.dev_04_w_static_task       dev
 13. Reference Set Report                             0.1.0    pbsmrtpipe.pipelines.dev_diagnostic             dev,reference,reports
 14. Reference Set Report + Random tasks              0.1.0    pbsmrtpipe.pipelines.dev_diagnostic_stress      stress,dev,reference,reports
 15. SubreadSet Reports and validation                0.1.0    pbsmrtpipe.pipelines.dev_diagnostic_subreads    subreads,pbvalidate,dev,reports
 16. Verify Chemistry Bundle Version                  0.1.0    pbsmrtpipe.pipelines.dev_verify_chemistry       subreads,dev
 17. Verify Sample Name Propagaton                    0.1.0    pbsmrtpipe.pipelines.dev_verify_sample_names    subreads,dev
 18. Verify chunking                                  0.1.0    pbsmrtpipe.pipelines.dev_subreads_chunk         subreads,dev

```

**Comment by mpkocher** [comment]
@zachcp  Unfortunately, we are not currently mirroring the https://github.com/PacificBiosciences/pbpipeline-resources to GitHub from our internal bitbucket system. 

I've filed an issue internally (SE-1105).

**Comment by mhsieh** [comment]
I am picking up SE-1105 now.

**Comment by zachcp** [comment]
Thanks guys.

**Comment by mhsieh** [comment]
the repo is updated

**Comment by zachcp** [comment]
will this be picked up automatically when building pbsmrtpipe with pitchfork?

**Comment by mhsieh** [comment]
we would need help from @mpkocher

**Comment by mhsieh** [comment]
Hi the best way to get the complete software is still getting it from http://www.pacb.com/support/software-downloads/ , which should provide enough information for this issue, so I am going to close this. If you found that you still need our assistance to unblock your use case, feel free to open a new issue.
